Gemfile

```
@@ -39,7 +39,7 @@ gem 'roadie-rails', '~> 3.0'
 gem 'deacon', '~> 1.0'
 gem 'mail', '~> 2.7'
 gem 'sshkey', '~> 2.0'
-gem 'dynflow', '>= 1.6.5', '< 2.0.0'
+gem 'dynflow', git: 'https://github.com/adamruzicka/dynflow.git', branch: 'kwargs-extravaganza'
```

There's also the `github` short hand if you want to type even less ;)
```suggestion
gem 'dynflow', github: 'adamruzicka/dynflow', branch: 'kwargs-extravaganza'
```

---

til